src/charts/row.js

```diff
@@ -961,12 +952,10 @@ define(function(require) {
                 const ltc = labelsTotalCount.toLocaleString();
                 const compCountTxt = labelsTotalText + ' ' + ltc;
                 let cw = textHelper.getTextWidth( compCountTxt, labelsSizeChild, 'Karla, sans-serif');
-                let printPadding = isPrintMode && isIE ? 10 : 0;
```

Comment:
`printPadding` is still used on line 972

<img width="566" alt="Screen Shot 2022-06-13 at 11 09 19 AM" src="https://user-images.githubusercontent.com/704760/173385382-2a23c974-6475-4638-9e1f-e9aab7354979.png">